cobbler.spec

```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
 # systemd
 mkdir -p %{buildroot}%{_unitdir}
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd.service %{buildroot}%{_unitdir}
+ln -sf service %{buildroot}%{_sbindir}/rccobblerd
 
 # cobbler-web
 rm %{buildroot}%{_sysconfdir}/cobbler/cobbler_web.conf
 
+# Remove duplicates
+%fdupes -s %{buildroot}%{_prefix}
+
 
 %pre
 %if %{_vendor} == "debbuild"
 if [ "$1" = "upgrade" ]; then
 %else
 if [ $1 -ge 2 ]; then
+    %if 0%{?suse_version}
+    %service_add_pre cobblerd.service
+    %endif
```

Comment:
This doesn't do anything useful, we don't have SysV scripts to migrate from and haven't had them for a long time.

Reply:
Correct me if I am wrong but if I understood this wiki article correctly it does more than that: https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines#Register_systemd_unit_files_in_install_scripts

Reply:
Hm okay. Then I will revert this. The linter complained and now doesn't so I will add this to the rpmlintrc along with the service calls below.